src/test/java/org/broadinstitute/hellbender/engine/spark/JoinReadsWithRefBasesSparkUnitTest.java

```diff
@@ -28,7 +27,7 @@
     @DataProvider(name = "bases")
     public Object[][] bases(){
         Object[][] data = new Object[2][];
-        List<Class<?>> classes = Arrays.asList(Read.class, SAMRecord.class);
+        List<Class<?>> classes = Arrays.asList(SAMRecord.class, SAMRecord.class);
```

Comment:
This should be a singleton list now, at least until there is no other implementation.

Reply:
oh, good call, I just replaced read -> samrecord, I wasn't paying close enough attention here

Reply:
opening a new ticket to simplify these if we're only going to have 1 read type #4318